<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newTeacherDialog</class>
 <widget class="QDialog" name="newTeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fioLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Кафедра</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Должность</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="phoneLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="facultyComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="postComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="selectRecordButton">
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="canselButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
